nifi-nar-bundles/nifi-enrich-bundle/nifi-enrich-processors/src/main/java/org/apache/nifi/processors/AbstractEnrichIP.java

```diff
@@ -70,6 +70,16 @@ public abstract class AbstractEnrichIP extends AbstractProcessor {
             .addValidator(StandardValidators.createAttributeExpressionLanguageValidator(AttributeExpression.ResultType.STRING))
             .build();
 
+    public static final PropertyDescriptor LOG_LEVEL = new PropertyDescriptor.Builder()
+            .name("log-level")
+            .displayName("Log Level")
```

Comment:
Recommend using the same name and displayName values.
```suggestion
            .name("Log Level")
            .displayName("Log Level")
```